src/mame/drivers/r9751.cpp

```diff
@@ -48,6 +48,7 @@
 ******************************************************************************/
 
 /* Core includes */
+#include <queue>
```

Comment:
`#include` order should be PCH (emu.h), module header, project headers, framework headers, and standard library headers in that order.  This helps avoid a lot of pitfalls.  Please don't `#include` anything before the PCH in particular.

Reply:
I'll fix that.